css/main.css

```diff
@@ -168,40 +161,82 @@ kbd {
 }
 
 .contentContainer{
+	/*inner stuff*/
 	display: flex;
-	min-width: 0;
-	min-height: 0;
+	margin: 0 0.5em;
+}
+
+.fullscreenBody .contentContainer{
+	/* on contentContainers inside a fullscreenBody, scroll the contentContainer instead of the page.*/
+	flex: 1;
 	overflow: hidden;
-	flex: 1; /* contentContainer fills the remaining empty space below the flexTopBar */
 }
 
-.sidebarNirnroute{
+.resizer{
+	background-color: black;
+	cursor: ew-resize;
+	padding: 2px;/*actual content of resizer*/
 	margin-left: 0.5em;
-	user-select: none;
+	/*on fullscreen pages, don't scroll with the rest of the content*/
+	position:sticky;
+	top:0
+}
+
+#mapColumn{
+	position:sticky;
+	top:0;
+}
+
+.mainPanel{
+	min-width: 10%;
+}
+
+.sidePanel{
+	flex: 1 1 0%;
+	min-width: 10%;
+	border: 1px solid black;
```

Comment:
border is ugly, much nicer looking to have no border and have the content touching the edges of the screen via no contentContainer margins.

Reply:
If the resize divider is part of the  visual "right panel", the right panel needs to have borders on the other sides indicating the borders of that panel.

Reply:
I guess I'm confused about your description of it. To an end user, the divider is just a bar that separates two halves of the screen, they won't see the bar as part of either particular side since it resizes them both equally. At least the way I see it, if one panel doesn't have the border, the other shouldn't either, just looks cleaner having everything the same and both sides touching all the way to the edge of the screen imo.